server/scripts/populate.py

```diff
@@ -0,0 +1,150 @@
+    club_list = read_clubs(filepath)
+    if club_list is None:
+        print("Something failed; aborting...")
+        return
+    conn = connect()
+    if conn is None:
+        return
+    insert_tags()
+    try:
+        while True:
+            club = next(club_list)
+            club["id"] = str(uuid.uuid4())
+            # pp.pprint(club)
+            with conn.cursor() as cursor:
+                cursor.execute("INSERT INTO club VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s);",
+                               (club.get("id"), datetime.now(), datetime.now(), club.get("name"),
+                                club.get("description"), club.get("size"), club.get(
+                                    "banner_image"), club.get("icon_image"),
+                                club.get("facebook"), club.get("twitter"), club.get("instagram"), club.get("website")))
+                events = club.get("events")
+                if (events is not None):
+                    for event in events:
+                        event["id"] = str(uuid.uuid4())
+                        cursor.execute(
+                            "INSERT INTO event VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s);",
+                            (event.get("id"), datetime.now(), datetime.now(), event.get("name"), event.get("description"),
+                             event.get("url"), event.get("banner_image"),
+                             event.get("start_time"), event.get("end_time"),
+                             event.get("facebook"), event.get("twitter"), event.get("instagram")))
+                        cursor.execute(
+                            "INSERT INTO club_events VALUES (%s, %s);", (club.get("id"), event.get("id")))
+                        tags = event.get("tags")
+                        if tags is not None:
+                            for tag in tags:
+                                cursor.execute(
+                                    "INSERT INTO tag_events VALUES (%s, %s);", (tag_entities.get(tag), event.get("id")))
+                tags = club.get("tags")
+                if tags is not None:
+                    for tag in tags:
+                        cursor.execute(
+                            "INSERT INTO tag_clubs VALUES (%s, %s);", (tag_entities.get(tag), club.get("id")))
+    except StopIteration:
+        conn.commit()
+        print("Clubs Inserted")
+    finally:
+        conn.close()
+        print("Connection Closed")
+
+
+def connect():
+    conn = None
+    print("Connecting to PostgreSQL server...")
+    if not path.exists("db.ini"):
+        print("Missing configuration file.")
+        return None
+    parser = ConfigParser()
+    parser.read("db.ini")
+    try:        
+        conn = psycopg2.connect(
+            host=parser["postgresql"].get("host"),
+            database=parser["postgresql"].get("database"),
+            user=parser["postgresql"].get("user"),
+            password=parser["postgresql"].get("password"))
```

Comment:
Accessing a dictionary by string key is actually quite slow. `parser["postgresql"]` should be stored in an intermediate variable so that it's only called once